tck-download/jakarta-expression-language-tck/pom.xml

```
@@ -0,0 +1,85 @@
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+>
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>fish.payara.jakarta.tests.tck</groupId>
+        <artifactId>tck-download</artifactId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>jakarta-expression-language-tck</artifactId>
+    <packaging>pom</packaging>
+    <name>TCK: Install Jakarta Expression Language TCK</name>
+
+    <properties>
+        <tck.test.expression-language.file>jakarta-expression-language-tck-${tck.test.expression-language.version}.zip</tck.test.expression-language.file>
+        <tck.test.expression-language.url>https://download.eclipse.org/jakartaee/expression-language/6.0/${tck.test.expression-language.file}</tck.test.expression-language.url>
+        <tck.test.expression-language.version>6.0.0</tck.test.expression-language.version>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>com.googlecode.maven-download-plugin</groupId>
+                <artifactId>download-maven-plugin</artifactId>
+                <version>1.9.0</version>
+                <configuration>
+                    <url>${tck.test.expression-language.url}</url>
+                    <unpack>true</unpack>
+                    <outputDirectory>${project.build.directory}</outputDirectory>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>download-expression-language-tck</id>
+                        <goals>
+                            <goal>wget</goal>
+                        </goals>
+                        <phase>generate-resources</phase>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <plugin>
+                <artifactId>maven-install-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>install-expression-language-tck-pom</id>
+                        <goals>
+                            <goal>install-file</goal>
+                        </goals>
+                        <phase>process-resources</phase>
+                        <configuration>
+                            <file>${project.build.directory}/el-tck/artifacts/jakarta-expression-language-tck-${tck.test.expression-language.version}.jar</file>
+                            <sources>${project.build.directory}/el-tck/artifacts/jakarta-expression-language-tck-${tck.test.expression-language.version}-sources.jar</sources>
+                            <groupId>jakarta.tck</groupId>
+                            <artifactId>expression-language-tck</artifactId>
+                            <version>${tck.test.expression-language.version}</version>
+                            <pomFile>src/resources/expression-language-tck-6.0.0.pom</pomFile>
```

Not sure we need this? We can just generate one
```suggestion
                            <generatePom>true</generatePom>
```